lang/translations/base.en_US.json

```diff
@@ -0,0 +1,4 @@
+{
+  "settings.date.format": "01/02/2006",
```

Comment:
should these settings be loaded from a different file that is just associated with region, to avoid the combinatorial explosion of language * region? (e.g. es_US, fr_CA, etc)

Reply:
That is an interesting thought - do other localisation systems work that way? I thought the lookup was strictly hierarchical, i.e. en_GB_Latn falls back to en_GB falls back to en?

Reply:
Not sure, but when testing it with my system set to Spanish (es_US), Monday was the first day of the calendar

Reply:
I would suggest they could even start as hard-coded before we think of a better place to configure them (and if they even need to be configured in a text-based metadata file)